src/arrays/practice/1.js

```diff
@@ -1,9 +1,35 @@
-function arrayManipulations() {
-  // your code here
-  let arr = [1,2,3];
-  arr.filter((elem, i) => {
-      console.log(elem);
-  });
+function arr1(arr) {
```

Comment:
Rename function name to something more verbose

Reply:
It was just for me